src/util/jpeg_resolution.cc

```diff
@@ -59,14 +61,14 @@ using uchar = unsigned char;
 #define M_DRI 0xDD
 
 #define ITEM_BUF_SIZE 16
-static int get16m(const void* shrt)
+static int get16m(std::byte* shrt)
 {
-    return (static_cast<uchar*>(const_cast<void*>(shrt))[0] << 8) | static_cast<uchar*>(const_cast<void*>(shrt))[1];
+    return std::to_integer<int>((shrt[0] << 8) | shrt[1]);
 }
 
 static int iohFgetc(const std::unique_ptr<IOHandler>& ioh)
 {
-    uchar c[1] = { 0 };
+    std::byte c[1] {};
```

Comment:
Does this init `c[0]` to 0 (see return statement)?

Reply:
Should be identical. int() is equivalent to std::to_integer<int>